### PR TITLE
[MM-52919] Remove all uses of window.desktop.version from webapp

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
This PR cleans up the use of `window.desktop.version`, which was used for versions of the Desktop App before v5 to check for the version to turn functionality on or off.

As it stands, there's nothing that has used this since v5, and since v4.7 and before is no longer supported, removing this makes sense.

Going forward, we're going to implement a new Desktop API in the Web App to check to see if certain features are enabled or not.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-52919

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/logged_in/logged_in.tsx [hunk 1]
```diff
@@ -174,6 +175,7 @@ export default class LoggedIn extends React.PureComponent<Props> {
 
         switch (desktopMessage.data.type) {
         case 'register-desktop': {
+            // Currently used by calls
```
Comment (cpoile) [inline webapp/channels/src/components/logged_in/logged_in.tsx:178]: 👍

**Review by cpoile — APPROVED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by M-ZubairAhmed** [comment]
I have mentioned another community pr which was again checking this desktop version and asked to remove it

**Pull request merged**